HomeTask50/HomeTask50/YandexHomePage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System.Threading.Tasks;
+
+namespace HomeTask50
+{
+     public class YandexHomePage
+    {
+        private string writeMailButton = "//span[text()='Написать']/parent::a";
+        private string avatarImage = "//img[@class='user-pic__image']";
+        private IWebDriver Driver { get; set; }
+        private double Time;
```

Comment:
I think that you don't need a separate variable to set timeout period for implicit waiter. Just pass it directly to TimeSpan.FromSeconds method

Reply:
Looked at all the code fixed this problem.